### PR TITLE
DID pkg test coverage

### PR DESCRIPTION
**Pull request by decentralgabe**
Increasing test coverage. Minor refactors.

### CODE REVIEW

**Comment by decentralgabe** [comment]
actually found a few issues!

File: did/key.go [hunk 1]
```diff
@@ -31,6 +31,23 @@ const (
 	DIDKeyPrefix = "did:key"
 )
 
+func (d DIDKey) IsValid() bool {
+	return true
```
Comment (nitro-neal) [inline did/key.go:35]: no checks here?

**Review by nitro-neal — COMMENTED** [review]
looks great

File: did/key.go (hunk 1 above)
Comment (nitro-neal) [inline did/key.go:35]: no checks for this?

**Review by nitro-neal — COMMENTED** [review]
look great!

**Review by nitro-neal — APPROVED** [review]

File: did/key.go (hunk 1 above)
Comment (decentralgabe) [reply did/key.go]: updated, whoops

**Pull request merged**